snakemake_interface_software_deployment_plugins/__init__.py

```
@@ -306,6 +323,7 @@ def _managed_generic_hash(self, kind: str) -> str:
         if store is None:
             record_hash = f"record_{kind}"
             hash_object = hashlib.md5(usedforsecurity=False)
+            breakpoint()
```

Ruff (0.15.15)

[error] Trace found: `breakpoint` used

(T100)